HTML/3. lists.md

```diff
@@ -0,0 +1,38 @@
+# Lists
+
+There are two types of lists, unordered lists and ordered lists. Unordered lists are noted with the `<ul>` element , and ordered list are noted with the `<ol>` element.
+___
+For each of those elements you need an `<li>` element for each list item inside
+___
+
+Here's an example with the `<ul>` element
+
+```html
+<ul>
+	<li>Eat</li>
+	<li>Sleep</li>
+	<li>Code</li>
+</ul>
+```
+
+This would output:
+
+![Unordered List Picture](images/ul.png)
+
+___
+
+Here's an example with an `<ol>`
+
+```html
+<ol>
+	<li>Eat</li>
+	<li>Sleep</li>
+	<li>Code</li>
+</ol>
+```
+
+Output:
```

Comment:
Again, I would tell you to use sub/sub-sub headings before the code blocks and the outputs, but it is your wish